server/routes/wishCards.js

```diff
@@ -510,23 +511,33 @@ router.post('/unlock/:id', async (req, res) => {
   try {
     const {
       wishCardId,
-      lockedWishcard,
+      alreadyLockedWishCard,
       userId,
       error,
     } = await WishCardController.getLockedWishCards(req);
 
     if (error) handleError(res, 400, error);
 
-    if (lockedWishcard && lockedWishcard.isLockedBy === userId) {
-      await WishCardRepository.unLockWishCard(wishCardId);
-      io.emit('unblock', { id: wishCardId });
-      clearTimeout(blockedWishcardsTimer[wishCardId]);
+    console.log(alreadyLockedWishCard, userId)
+    if (alreadyLockedWishCard) {
+      if (`${alreadyLockedWishCard.isLockedBy}` === `${userId}`) {
```

Comment:
use the String() constructor here -> String(alreadyLockedWishCard) === String(userId)